numpyro/distributions/distribution.py

```diff
@@ -150,6 +150,29 @@ def event_shape(self):
         """
         return self._event_shape
 
+    @property
+    def event_dim(self):
```

Comment:
I would suggest creating separate PRs for non-brainer changes like these, so that we can merge it asap. :) Don't worry about it for this one though.